printer/printer.go

```diff
@@ -0,0 +1,59 @@
+package printer
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"text/template"
+)
+
+const (
+	FORMAT_PLAIN = "plain"
+	FORMAT_JSON  = "json"
+)
+
+type Printer struct {
+	Format string
+	Single bool
+	Lines  []interface{}
+}
+
+var printer Printer
+
+func SetFormat(t string) {
+	printer.Format = t
+}
+
```

Comment:
Would love it if you could add some godocs to the functions in this file. Reason I asked is because I didn't understand what `SetSingle()` meant from the function name alone.